client/src/components/FieldHolder/FieldHolder.js

```diff
@@ -53,7 +77,7 @@ function fieldHolder(Field) {
     /**
      * Build title label
      *
-     * @returns {Component}
+     * @returns {React}
```

Comment:
Shouldn't it be `Component` instead? Is `React` a type?

Reply:
React is the javascript class, which is what happens here... I'm still exploring this, might be more valid to be `Symbol` I think, but `Component` isn't correct.
JSDocs take `Component` as HTML Web Component iirc